ui/scripts/vpc.js

```diff
@@ -1333,35 +1321,65 @@
+                                                    var items = json.listnetworkaclsresponse.networkacl;
+
+                                                    if(items){
+                                                        items.sort(function(a, b) {
+                                                            return a.number >= b.number;
+                                                        }).map(function(acl) {
+                                                            if (parseInt(acl.protocol)) { // protocol number
+                                                                acl.protocolnumber = acl.protocol;
+                                                                acl.protocol = "protocolnumber";
+                                                            }
+    
+                                                            return acl;
+                                                        });
+                                                    }
 
                                                     args.response.success({
                                                         data: items
-                                                        /* {
-                               cidrlist: '10.1.1.0/24',
-                               protocol: 'TCP',
-                               startport: 22, endport: 22,
-                               networkid: 0,
-                               traffictype: 'Egress'
-                               },
-                               {
-                               cidrlist: '10.2.1.0/24',
-                               protocol: 'UDP',
-                               startport: 56, endport: 72,
-                               networkid: 0,
-                               trafficType: 'Ingress'
-                               }
-                               ]*/
                                                     });
+                                                    if(jQuery('#details-tab-aclRules').siblings('div.toolbar').children('div.add').size() === 0){
+                                                        var $addAclRuleDivButton = jQuery('<div>').addClass('button add');
+                                                        var $spanAddAclRuleButtonMessage = jQuery('<span>').html(_l('label.add.ACL'));
+                                                        
+                                                        $addAclRuleDivButton.html($spanAddAclRuleButtonMessage);
+                                                        $addAclRuleDivButton.click(function(){
+                                                        	cloudStack.dialog.createForm({
+                                                        		 form: {
+                                                                     title: 'label.add.rule',
+                                                                     desc: 'Create a new ACL rule',
```

Comment:
How about using internationalization here as well?

Reply:
Sure, I was maintaining everything as before, but why not improve this one as well.
I will do that in a sec.